### ISSUE TITLE
Ability to tab to and inside slides for accessibility

### ISSUE THREAD

**Issue by bethanyfulton** [blaze-slider/blaze-slider#81]
Really enjoying playing with this, but being based in Europe to use it in projects it needs to be accessible to those using screenreaders. Currently I can't figure out a way to tab to the individual slides, and therefore to any content inside of them. If I'm missing something and this is already possible, apologies!

**Comment by MananTank** [comment]
I'm not fully sure what you mean by "tab to the individual slides" - Can you give more details on the behavior you're expecting?

If you want to change slides on tab keydown, you can implement that using these methods - https://blaze-slider.dev/docs/api/methods

**Comment by MananTank** [comment]
@bethanyfulton I'm not sure the exact UX you're describing - Can you share an example?
